README.md

```diff
@@ -227,15 +227,29 @@ Any *backwards incompatible* change to either the Node.js Core Library API or Ap
 
 Issue: Should any modification to the ABI or Dependencies that requires module or application developers to recompile force a *semver-major* change?
 
+Node.js implements a number of default values and assumed behaviors. Some of these may be intended for module and application developers to take advantage of while others may not. As a general rule, if a given default value or assumed behavior is *documented* within the end-user API documentation, it should be considered part of the public API. When modifying such defaults, Collaborators should use their best judgement to determine whether any given change is "technically backwards incompatible but in practice should not be".
+
 APIs and default behaviors in the Node.js Core Library, Application Binary Interface, Dependencies and NAN must not change within LTS Releases unless the change is required to address a critical security update.  
 
-Default values or assumed behaviors that could cause modules or applications to break if modified are considered part of the API. This is a gray area, however.
+### Platform Stability
+
+The Platforms supported by the Node.js project are divided in four distinct categories: *Primary*, *Secondary*, *Experimental* and *Deprecated*.
```

Comment:
Ok, so a progression... *Primary* is anything in an LTS Release, *Secondary* is anything in *master* but may not necessarily have been in LTS yet.